frontend/src/metabase/query_builder/components/filters/pickers/DatePicker/RangeDatePicker.tsx

```diff
@@ -1,82 +1,106 @@
-/* eslint-disable react/prop-types */
-import React from "react";
-
-import Calendar from "metabase/components/Calendar";
-
-import moment from "moment-timezone";
+import React, { useCallback, useState } from "react";
+import moment, { Moment } from "moment-timezone";
 import Filter from "metabase-lib/lib/queries/structured/Filter";
-import { TimeContainer } from "./RangeDatePicker.styled";
-import {
-  hasTimeComponent,
-  setTimeComponent,
-  TIME_SELECTOR_DEFAULT_HOUR,
-  TIME_SELECTOR_DEFAULT_MINUTE,
-} from "metabase/lib/query_time";
+import { setTimeComponent } from "metabase/lib/query_time";
+import Calendar from "metabase/components/Calendar";
 import SingleDatePicker, { SingleDatePickerProps } from "./SingleDatePicker";
 import SpecificDatePicker from "./SpecificDatePicker";
+import { DateContainer, DateDivider } from "./RangeDatePicker.styled";
 
-type BetweenPickerProps = {
+export interface BetweenPickerProps {
   className?: string;
+  filter: Filter | any[];
   primaryColor?: string;
-  filter: Filter;
-  onFilterChange: (filter: any[]) => void;
-
   hideTimeSelectors?: boolean;
-};
+  onFilterChange: (filter: any[]) => void;
+}
 
 export const BetweenPicker = ({
   className,
   filter: [op, field, startValue, endValue],
-  onFilterChange,
-  hideTimeSelectors,
   primaryColor,
+  hideTimeSelectors,
+  onFilterChange,
```

Comment:
Looks to me that this is just a props reordering. Not sure if this does anything.